terraform/spanner/bind/main.tf

```diff
@@ -0,0 +1,50 @@
+locals {
```

Comment:
I think it is best to declare the locals in the `data` module. I don't know how keeping it here would play out with subsume, but most importantly, consistency wise we are declaring locals in the data module in the azure-brokerpak and in other modules in this brokerpak as well.